### DISCUSSION TITLE
Multiple library/database support

### DISCUSSION THREAD

**Discussion by busywait** [music-assistant/support#1008] [Feature requests and ideas]
In my Home Assistant /media directory I have audiobooks, music lessons (like exam grade accompaniments), and a general music library. They are in different subdirectories, /media/audiobooks, /media/lessons, and /media/music. I'd like to be able to choose which one I'm browsing because they are each quite large and the different types of audio distract me when I'm browsing.

In the past (using Twonky as a UPnP server) I've used genre tags for each (audiobooks, lessons, pop), and browsed Artists > Albums via genre, but Music Assistant doesn't offer that. It would be great if I had a way to have multiple Music Assistant instances for each category, or a quick way to switch between databases. Is that possible already?

**Comment by OzGav** [comment]
This has been implemented in MA 2.7 and above by using the provider filter in the respective view

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
